src/main/kotlin/gg/skytils/skytilsmod/mixins/hooks/renderer/TileEntityChestRendererHook.kt

```diff
@@ -35,7 +35,7 @@ fun setChestColor(
     ci: CallbackInfo
 ) {
     if (te.pos == ThreeWeirdosSolver.riddleChest) {
-        ticTacToeSolverColor.bindColor()
+        Skytils.config.threeWeirdosSolverColor.bindColor()
```

Comment:
This should just import the static and not the surrounding class.

Reply:
I'm going to try that out, thank you for the advice! Also, are Kotlin mixins viable for other… mods?

Reply:
> This should just import the static and not the surrounding class.

That doesn't appear possible. I'll revert it to the old method